### ISSUE TITLE
3.18.1 not working for username with space in it: no files are shown

### ISSUE THREAD

**Issue by Wikinaut** [nextcloud/android#9612]
The new version does not work: the client, upgraded to 3.18.1 shows "No files".

**Comment by AndyScherzinger** [comment]
cc @AlvaroBrey

**Comment by AlvaroBrey** [comment]
Cannot reproduce :confused:  I have tested with usernames with spaces, both in fresh 3.18.1 installs, and with an upgrade path 3.17.1 -> 3.18.1. In both cases the app works as expected.

@Wikinaut can you give more details? Server version, device, Android version, etc.

**Comment by Wikinaut** [comment]
Okay, for that specific user I checked their setting and found the reason, why no files where show: the flag for "file acces" ("Dateisystem-Zugriff") was deactivated. I think I did this while the 3.18.0 was not working/loading files from the server to prevent any damages until a new version became available.

Now I actived the Datei
![grafik](https://user-images.githubusercontent.com/1151915/147420101-a8473fc2-3de8-4443-aea5-4d87d3e7ade3.png)
and the synchronisation / file lists is recreated.

Therefore I close this issue now, let's say as "invalid". I confirm, the 3.18.1 _is_ working for a user with a space in their useraccount name.
